### ISSUE TITLE
Improving fix options

### ISSUE THREAD

**Issue by FStefanni** [clangd/clangd#1066]
Hi,

I believe it could be useful to show these options, when showing a fix proposal:

1. Perform the single fix, as currently done
2. Perform all the fixes of the same type in the current file
3. Perform all possible fixes in the current file

I find this very useful, and I use these options frequently in other languages (e.g. eslint fixes with typescript).

Regards

**Comment by sam-mccall** [comment]
Agree, this is a dup of https://github.com/clangd/clangd/issues/830.
